src/pages/HowItWorks.tsx

```diff
@@ -18,20 +17,26 @@ export default function HowItWorks() {
   return (
     <>
       <div className="flex flex-col items-center justify-center">
-        <div className="flex max-w-[1440px] flex-col items-center justify-center ">
+        
+      <Image src={howItWorks1} alt='background' className="absolute top-0 left-0 w-full h-full object-cover z-0 "/> 
```

Comment:
In general, I think it's good practice to use a more descriptive text to describe the image for the `alt` attribute